openml/datasets/dataset.py

```diff
@@ -96,7 +96,7 @@ class OpenMLDataset(OpenMLBase):
         which maps a quality name to a quality value.
     dataset: string, optional
         Serialized arff dataset string.
-    minio_url: string, optional
+    parquet_url: string, optional
         URL to the MinIO bucket with dataset files
```

Comment:
I think we should also change the documentation here as the parquet file might not necessarily be hosted on a MinIO bucket.

Reply:
Okay will look into this